main.go

```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"context"
 	"fmt"
+	"github.com/urfave/cli/v2"
```

Comment:
Why not just run the standard package?

Reply:
updated my comment. gqlgen itself also uses urfave, so I thought it might be a good idea to use it for gqlgenc as well, for consistency. also we might start adding more command-line customization and it would be more convenient to use urfave comparing instead of standard package. if you disagree - I could rewrite it using a standard package, but I don't think that we have too much overhead by using urfave.